packaging/linux/run_keybase

```diff
@@ -147,7 +152,7 @@ start_systemd() {
   )
   forward_vars "$runtime_dir/keybase.gui.env" forwarded_gui_env_vars[@]
 
-  keybase-mount-helper -user-mount "$mountdir"
+  run_redirector
```

Comment:
It'd be nice to have a systemd service for the redirector, since all other services have.